app/controllers/administrateurs/groupe_instructeurs_controller.rb

```diff
@@ -100,6 +103,7 @@ def create
         .new({ instructeurs: [current_administrateur.instructeur] }.merge(groupe_instructeur_params))
 
       if @groupe_instructeur.save
+        @groupe_instructeur.procedure.toggle_routing
```

Comment:
```suggestion
        procedure.toggle_routing
```

Ca allège le code et ça évite peut être une requete en plus (?)